ocaml/xapi/xapi_dr_task.ml

```diff
@@ -31,7 +31,7 @@ type sr_probe_sr = {uuid: string; name_label: string; name_description: string}
 (* Attempt to parse a key/value pair from XML. *)
 let parse_kv = function
   | Xml.Element (key, _, [Xml.PCData v]) ->
-      (key, String.strip String.isspace v) (* remove whitespace at both ends *)
+      (key, String.trim v)
```

Comment:
Is it a problem that we are trimming these `\x0C' (form feed), '\n', '\r', and '\t' as well as the space?

Reply:
isspace is a misnomer, it should be called is_whitespace:

it's defined as:
```ocaml
let isspace = function ' ' | '\n' | '\r' | '\t' -> true | _ -> false
```